src/common/servlist.c

```diff
@@ -359,6 +359,9 @@ static const struct defaultserver def[] =
 
 	{"Xertion", 0, 0, 0, LOGIN_SASL, 0, TRUE},
 	{0,			"irc.xertion.org"},
+	
+	{"DeltaPool",		0},
```

Comment:
Update this to include that it supports SASL and SSL like other entries.

Reply:
Done!